README.md

```diff
@@ -10,20 +10,13 @@
 A generalized version of [*iOS Dark Mode Theme*](https://github.com/basnijholt/lovelace-ios-dark-mode-theme)!
 This includes both **Dark and Light Mode** and 7 different HomeKit backgrounds.
 Installing this theme adds 28 different themes:
```

Comment:
28 you say...

Reply:
Are you counting up to some different number?